bookwyrm/views/status.py

```diff
@@ -176,7 +176,7 @@ def post(self, request, status_id):
 
         # perform deletion
         status.delete()
-        return redirect("/")
```

Comment:
If you delete a status from the status' own page, I think this is going to redirect you to a 404?

Reply:
I reverted this change. I'd still like to look at this, but I'll do that in a different PR as I don't have time for this at moment.